### PR TITLE
KIN-370: Privacy Policy Page Layout

### PR DESCRIPTION
**Pull request by CarlosJCD**
[KIN-370](https://loopcrack.atlassian.net/browse/KIN-370) Medium [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem

A dedicated page for the privacy policy of the website.

### Solution

Implementation of the privacy policy page.

### Requirements checklist

- [x] The privacy policy link must redirect to the privacy policy page
- [x] The privacy policy page must contain a title and a placeholder text

### Testing

- [x] Click the `avisos de privacidad` link of the footer to get redirected to the privacy policy page.

## Demo

Screenshot 2023-12-16 at 0 22 15

[KIN-370]: https://loopcrack.atlassian.net/browse/KIN-370?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Cristianpan — APPROVED** [review]
For devices smaller than 768px, it has a very large x padding, a possible solution would be to give the container a width of 80%. Please make the change before merging.

CC: @CarlosMay7 

image

**Review by CarlosMay7 — APPROVED** [review]
Nice work! Just check the nitpick @Cristianpan left you

File: src/public/sass/ITCSS/06_components/pages/privacy-policy/_privacy-policy.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@use '../../../02_tools/mixins' as *;
+
+.privacy-policy {
+  &__heading {
+    color: $color-aqua-blue;
+  }
+
+  &__text {
+    color: $color-gray-50;
+    font-size: 1.2rem;
```
Comment (Mauriciocr207) [inline src/public/sass/ITCSS/06_components/pages/privacy-policy/_privacy-policy.scss:14]: The `font-size:1.2rem` make the font a little big. I think 1rem is good
```suggestion
    font-size: 1rem;
```

**Review by Mauriciocr207 — APPROVED** [review]

**Review by CheloChidoone — APPROVED** [review]
Good Job @CarlosJCD, just finish checking Maus comments but I think it looks good

**Comment by CheloChidoone** [comment]
Additional comment @CarlosJCD change the route to be just `aviso` instead of `aviso_privacidad`, and if you're going to use two words for a url separate them with a `-` not a `_`

**Review by Oswaldo-Chan — APPROVED** [review]

**Review by FernandoJoachin — APPROVED** [review]

**Pull request merged**